crates/ef-testing/src/evm_sequencer/evm_state/v1.rs

```diff
@@ -209,7 +209,7 @@ impl Evm for KakarotSequencer {
         // Bytecode is stored in chunks of 31 bytes. At bytecode_base_address,
         // we store the number of chunks.
         let num_chunks = bytecode_len;
-        let mut bytecode: Vec<u8> = Vec::new();
+        let mut bytecode: Vec<u8> = Vec::with_capacity(bytecode_len as usize * 31);
```

Comment:
can this overflow?

Reply:
not realistically (2**31 bytes of EVM bytecode?)

Reply:
It could, but if it does, it means we are trying to read more than usize::MAX bytes in the vector and that will panic anyway if I remove this line